From 3a314210076d111771848978a7204fd14733d4f5 Mon Sep 17 00:00:00 2001
From: Steven Laver <lavers@google.com>
Date: Thu, 2 Jan 2020 14:19:11 -0800
Subject: [PATCH 10/10] bluetooth: select bluetooth modules by fully qualfified
 namespace.

This corresponds to Android.bp changes in
packages/apps/Bluetooth:86dd33026b2daedce89ddbf20760e22a1989734c.

Bug: 140404592
Change-Id: Icbf678be4bb89fb30c214b1ad0dda390140aa7ce
---
 apex/Android.bp | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/apex/Android.bp b/apex/Android.bp
index fc143ba3f..82f871349 100644
--- a/apex/Android.bp
+++ b/apex/Android.bp
@@ -4,10 +4,10 @@ apex {
     manifest: "apex_manifest.json",
 
     native_shared_libs: [
-      "libbluetooth_jni",
+      "//packages/apps/Bluetooth:libbluetooth_jni",
       "libbluetooth"
     ],
-    apps: ["Bluetooth"],
+    apps: ["//packages/apps/Bluetooth:Bluetooth"],
 
     compile_multilib: "both",
 
-- 
2.28.0

